api/tests/integration/domain/usecases/create-pro-organization-with-tags_test.js

```diff
@@ -199,145 +316,164 @@ describe('Integration | UseCases | create-pro-organization', function () {
         email: 'youness@example.net',
         locale: 'fr-fr',
         tags: 'TagNotFound',
+        createdBy: userId,
+        documentationUrl: null,
+        organizationInvitationRole: null,
       },
       {
-        externalId: '',
+        externalId: 'b300',
         name: 'Andreia & Co',
         provinceCode: '345',
         credit: 10,
         email: 'andreia@example.net',
         locale: 'fr-fr',
-        tags: 'TagNotFound',
-      },
-      {
-        externalId: 'b201',
-        name: 'Mathieu Bâtiment',
-        provinceCode: '567',
-        credit: 20,
-        email: 'mathieu@example.net',
-        locale: 'fr-fr',
-        tags: 'TagNotFound',
+        tags: 'AnotherTagNotFound',
+        createdBy: userId,
+        documentationUrl: null,
+        organizationInvitationRole: null,
       },
     ];
 
     // when
     const error = await catchErr(createProOrganizationsWithTags)({
       domainTransaction,
-      organizations: organizationsWithTagsWithOneMissingExternalId,
+      organizations: organizationsWithTagsNotExists,
       organizationRepository,
       tagRepository,
       organizationTagRepository,
       organizationInvitationRepository,
     });
 
     // then
-    expect(error).to.be.instanceOf(ObjectValidationError);
-    expect(error.message).to.be.equal('L’externalId de l’organisation n’est pas présent.');
+    expect(error).to.be.instanceOf(OrganizationTagNotFound);
+    expect(error.message).to.be.equal("Le tag TagNotFound de l'organisation Mathieu Bâtiment n'existe pas.");
     const organizationsInDB = await knex('organizations').select();
     expect(organizationsInDB.length).to.equal(0);
     const organizationTagsInDB = await knex('organization-tags').select();
     expect(organizationTagsInDB.length).to.equal(0);
   });
 
-  it('should rollback create pro organizations with tags and throw an error when an organization name is missing', async function () {
+  it('should create pro organizations with tags when tags already exists  ', async function () {
```

Comment:
**question**
Je ne comprends pas trop ce cas de test : ça veut dire quoi "quand les tags existes déjà" ? 
Est-ce que ça ne devrait pas être juste : "when tag has been found" plutôt ? 
De plus je ne vois pas dans le `//given` la création des tags en BDD ... du coup je ne comprends pas ce qui est testé 🤔

Reply:
Ils sont créés dans le beforeEach. Si les tags n'existent pas en base, on peut pas attribuer à l'orga le tag qu'on a mis dans le csv pour cette orga. ça te paraît plus clair ?

Reply:
j'ai mis la création des tags dans les 2 tests qui en avaient besoin (même si c'était juste fixer le message d'erreur normalement cette pr ^^)

Reply:
ah ok merci pour la descente des infos du beforeEach c'était pour ça que je comprenais rien 🤯

Reply:
C'est un peu plus claire pour le contenu du test, par contre je trouve toujours le mot "already" confusant dans ce test : il me laisse penser que l'orga est déjà créée et a déjà ce tag et qu'on essaie de lui réattribuer à nouveau ce tag qui existe déjà ... 
Si je comprends bien ce test : on essaie de vérifier si les orga se crée bien avec les tags s'ils existent non ?
Donc je propose plutôt `should create pro organizations with tags when tags exists` ou `should create pro organizations with tags when tags has been found`

Reply:
le code crée les orgas à la volée et associes les tags ensuite mais ça throw si un tag n'existe pas en base. En vrai un fonction qui checkerait les tags du csv juste avant ça serait cool mais on remet 10 balles dans la machine ^^